build/root/Makefile

```diff
@@ -344,6 +344,26 @@ clean_generated:
 	find . -type f -name $(GENERATED_FILE_PREFIX)\* | grep -v "[.]/staging/.*" | xargs rm -f
 endif
 
+define CLEAN_CODEGEN_HELP_INFO
+# Remove generated code from hack/update-codegen.sh, including client-gen, lister-gen and informer-gen.
+#
+# Example:
+#   make clean_codegen
+endef
+.PHONY: clean_codegen
+ifeq ($(PRINT_HELP),y)
+clean_codegen:
+	@echo "$$CLEAN_CODEGEN_HELP_INFO"
+else
+clean_codegen:
+	grep -Rl "Code generated by client-gen. DO NOT EDI"   pkg/client/clientset_generated/          | xargs rm
```

Comment:
s/EDI/EDIT ?

Reply:
The "EDI" was used to make `hack/verify-boilerplate.sh` happy. I added comments about why we do this. 

Did a rebase on HEAD and verify-boilerplate still fails if we use "EDIT"